docs/graph-data-init-container.md

```diff
@@ -20,7 +20,7 @@ When the init container runs, it untars the data to /var/lib/cincinnati/graph-da
 Build and push the image to your own repository. 
 
 ````
-podman build -f ./dev/Dockerfile -t quay.io/rwsu/cincinnati-graph-data-container:latest
+podman build -f ./dev/Dockerfile --platform=linux/amd64 -t quay.io/rwsu/cincinnati-graph-data-container:latest
```

Comment:
Maybe fix the rwsu thing right away too and use a `your-registry/your-repo/cincinnati-graph-data-container:latest`  placeholder like you used in README?

Reply:
Done.